src/hardware/developing-an-iox/protocol.md

```diff
@@ -237,6 +238,7 @@ Sent from the IOX to the GO device when the IOX wants create a log that cannot f
 ### IOX Request/Status (0x25)
 
 Sent from the IOX to the GO device to inform the GO device of events or status changes.
+A message of this type is to be added before the start of a [0x0C](#rx-data-0x0c) message series, and also after the end of it.
```

Comment:
This may not be the best place for this comment. Message 0x25 contains several 'types', this comment is only applicable to 'Type 1 - Packet Wrapper'.

Reply:
I have made the changes, please let me know if this change would be the best place to comment.

Reply:
This looks like the right place. However, the rich diff shows the comment added as a new row to the table, but the markdown looks correct to me. Maybe @muyuncaigeotab might have more experience with markdown and could comment?

Reply:
Oh ! I think I managed to miss out on the enter button but I made a change let me know if that corrects the note: rather than adds it to a new table. 
It did not record any change to push to origin. 
A screen shot of the URL preview when using gulp is in the soleng Jira ticket.